wiki/Skinning/Interface/en.md

```diff
@@ -31,16 +31,17 @@ Notes:
     - If the background is transparent, the background color will be black.
 - The player can drag and drop an image to overwrite the skin's skinned image
   - **This will delete the skin's image in the process!**
+- Background images can only be 1366x768 and will be upscaled to the player's resolution.
```

Comment:
this isn't true, you have to read more carefully and/or test using your own game client before committing these changes

the forum post says (again, I'm not sure if forum post is true so you should check) that the game doesn't use the @ 2x variant of this, so the skinner should use the highest quality image available for the "SD" version

Reply:
The game does not use the 2x varient. Just tested.

Reply:
... so can you fix this part? not sure why you marked as resolved

Reply:
The note is gone now. Should I leave the HD Size as a - or add the 2x resolution?